sonar-plugin-api/src/main/java/org/sonar/api/measures/Measure.java

```diff
@@ -322,27 +322,22 @@ public Integer getIntValue() {
   /**
    * Sets the measure value with a given precision
    *
-   * @param v         the measure value
-   * @param precision the measure value precision
-   * @return the measure object instance
+   * @return {@code this}
+   * @deprecated in 5.3. The decimal scale is given by the metric, not by the measure. Anyway this parameter was enforced to 1 before version 5.3.
    */
-  public Measure<G> setValue(@Nullable Double v, int precision) {
+  @Deprecated
+  public Measure<G> setValue(@Nullable Double v, int decimalScale) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)